File: java/dayOne/levelOne/TaskTwo.java
import java.util.Arrays;

public class TaskTwo {

    public static boolean checkPalindrome(int [] numbers){
        
            
           int [] reversedArray = new int [numbers.length];
    
           int start = 0;
           int end = reversedArray.length-1;
        
            while(end >= 0){
                
                reversedArray[start] = numbers[end];

                start++;
                end--;
                
                }
                  
            
                int palindromeNumber = 0;
            for(int count = 0; count < numbers.length; count++){

                if(numbers[count] == reversedArray[count]){
                    palindromeNumber++;}
                  
                 }

                if(palindromeNumber == numbers.length)
                
                    return true; 
                
                else{
                   
                
                } return false;
                 
                
    }

}

File: python/day_one/level_one/task_two.py


def check_palindrome(numbers):

    reversed_list = []   
    
    for number in numbers[::-1]:
        reversed_list.append(number) 
    
    count = 0   
    for index in range(len(reversed_list)):                
        if(numbers[index] == reversed_list[index]):
            count = count + 1
    
    if(count == len(numbers)):
        return True
    else:
        return False    




